src/prob/common.jl

```diff
@@ -177,6 +177,7 @@ function solve_mc_model(
     make_si::Bool=!get(data, "per_unit", false),
     make_si_extensions::Vector{<:Function}=Function[],
     dimensionalize_math_extensions::Dict{String,Dict{String,Vector{String}}}=Dict{String,Dict{String,Vector{String}}}(),
+    transformations::Vector{<:Function}=Function[],
```

Comment:
If we keep this around in addition to eng2math_extensions, which its not clear yet to me that it is needed in addition to that, we should probably change the name to reflect the transformations happen on the math model. Also, a mention would need to be added to the docstring.